src/ir_Airwell.h

```diff
@@ -91,7 +94,7 @@ class IRAirwellAc {
   IRsendTest _irsend;  ///< Instance of the testing IR send class
   /// @endcond
 #endif
-  uint64_t remote_state;  // The state of the IR remote in native IR code form.
+  AirwellProtocol _;
   void checksum(void);
```

Comment:
We should probably remove `checksum(void)` as I don't think it exists.
(Not your problem, but while you're updating it! :-)